bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import datetime
+import logging
+from typing import List, Optional
+
+from discord import Color, Embed, Message
+from discord.ext import commands
+
+from bot.constants import Wikipedia
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    @staticmethod
+    def formatted_wiki_urls(index: int, titles: str) -> List[str]:
+        """Making formatted wikipedia links list."""
+        titles = f'`{index}` [{titles}]({WIKIPEDIA_URL.format(title=titles.replace(" ", "_"))})'
+        return titles
```

Comment:
I'm for the staticmethod change, but the naming doesn't quite add up. You're typing titles as a string, but titles implies its a list of titles. formatted_wiki_urls & List[str] - also implies it returns a list, but you're just returning a string.

I'd rewrite this function to only handle creating one line, and using loops/comprehensions with it.
```suggestion
    @staticmethod
    def formatted_wiki_url(index: int, titles: str) -> str:
        """Making formatted wikipedia link.."""
        return  f'`{index}` [{title}]({WIKIPEDIA_URL.format(title=title.replace(" ", "_"))})'
```